.pipeline-assets/pipeline-build-solutions.PS1

```diff
@@ -156,22 +158,25 @@ foreach ($folder in $deviceFolders)
     {
         "" | Write-Host -ForegroundColor Red
         "*****************************************" | Write-Host -ForegroundColor Red
-        ">>>ERROR: Couldn't find the nuspec file!"  | Write-Host -ForegroundColor Red
+        "##[error] >>>ERROR: Couldn't find the nuspec file!"  | Write-Host -ForegroundColor Red
         throw "Couldn't find the nuspec for '$folder'..."
     }
     else
     {
-        "INFO: nuspec file is: '$nuspecFile'" | Write-Host -ForegroundColor Yellow
+        "##[command] INFO: nuspec file is: '$nuspecFile'" | Write-Host -ForegroundColor Yellow
     }
 
     # need to restore NuGets first
+    write-host "##[command] Performing nuget restore for '$solutionFile'."
     nuget restore $solutionFile
     if (-not $?)
     { 
         "" | Write-Host -ForegroundColor Red
         "*****************************************" | Write-Host -ForegroundColor Red
-        ">>>ERROR: Couldn't restore solution '$solutionFile'!"  | Write-Host -ForegroundColor Red
-        throw "Error restoring '$solutionFile'"
+        "##[error] >>>ERROR: Couldn't restore solution: '$solutionFile'!"  | Write-Host -ForegroundColor Red
+        # throw "Error restoring '$solutionFile'"
```

Comment:
This is on purpose to decrease the number of pipeline runs.
If it throws the build won't fail and will move on the next solution. At the end all failing solutions can be found.
Then, one can fix all the failing solutions and submit the fixes in a single batch.
Otherwise, the pipeline will have to run for each fix.

Reply:
Fair enough. I was trying to find the cause of the current error to start with, so wanted to make sure I was not missing a silent error. The pipeline syntax highlighting and sections have helped, but still cannot track down the issue. I will revert this part when we are able to get to that state.

Reply:
The build is now OK.
It was an edge case with the nested dependency check on Units Net.
The build script here is doing just fine 😉